src/_community/pluralkit.md

```diff
@@ -0,0 +1,79 @@
+---
+title: PluralKit
+---
+
+{% admonition %}
+**Note:** This document is specific to Discord. If you're not making use of our community spaces on Discord, PluralKit
+won't be available.
+{% endadmonition %}
+
+[PluralKit](https://pluralkit.me) is a Discord bot that exists to allow users to proxy their messages via Discord webhooks. This has many practical applications, especially given that Discord doesn't have any features that support the plural community, or any built-in mental health aids.
+
+PluralKit has all kinds of fun and interesting uses, but in Quilt community spaces it should only be used for [plurality](#plurality) or self-identity purposes, or as a mental health aid. We do not allow users to make use of PluralKit for role-playing.
+
+If you see users talking with the bot tag, they're talking through PluralKit. You can react to any message sent this way with the `question` emote (❓) for information on which Discord account the message belongs to.
+
+## General Usage
+
+The primary function of PluralKit is automatically proxying messages that people using it send, in order to make those messages appear to come from someone else. This is done by making use of Discord webhooks - which has the unfortunate side effect of Discord stating that the messages were sent by a bot. Rest assured that this isn't the case - while the messages were proxied by PluralKit, they were written by another (human) member of the community
+
+As a member of the community, there's only one PluralKit feature available to you by default - the ability to figure out which Discord account was used to send a proxied message. You can do this by reacting to any proxied message using the `question` emote (❓), which will cause PluralKit to send you a private message with information about the Discord account who sent the message, such as username, nickname, and roles.
```

Comment:
This is no longer the case - 🔔 can be used to ping the account that sent the message as well.